src/login/login.resource.tsx

```diff
@@ -0,0 +1,20 @@
+export function performLogin(username, password) {
+  const token = window.btoa(`${username}:${password}`);
+  return window
+    .fetch(`/openmrs/ws/rest/v1/session`, {
```

Comment:
Making the API call to login to openmrs with the REST api

Reply:
@joeldenning is it possible to abstract this into a separate file?

Reply:
Will this work when the context is named anything other than 'openmrs'?

Reply:
@fatmali it already is in a separate file - `login.resource.tsx`

@dkayiwa no it won't, good catch. I'll change that.

Reply:
Going to merge this as-is and capture the need to account for other context's besides openmrs in https://github.com/openmrs/openmrs-esm-login/issues/2